cmd/rook/rgw.go

```diff
@@ -40,7 +40,6 @@ var (
 )
 
 func init() {
-	rgwCmd.Flags().StringVar(&rgwName, "rgw-name", "", "name of the object store")
```

Comment:
Did you test what happens if the flag is still set with this change? Does it make the pod fail because of an non-existent flag?

Reply:
I didn't, but I would expect that it would make the pod fail. Do you think it makes sense to leave it as an unrequired option for the change to be "less breaking"?

Reply:
As setting `&rgwName` to `nil` isn't possible (see https://play.golang.org/p/xKmcMD7hAvt).
Let's see what @travisn thinks, but yes overall leaving it in but as not required should be good.

Reply:
@dgonzalezruiz As you solved it now by simply setting the default value to `HOSTNAME` env var looks good.